cfspopcon/formulas/scrape_off_layer/lambda_q.py

```diff
@@ -29,6 +29,8 @@ def calc_lambda_q(
 ) -> float:
     """Calculate SOL heat flux decay length (lambda_q) from a scaling.
 
+    TODO: Remove in next major release, in favour of using algorithms directly.
+
```

Comment:
We should probably track this in an issue then

Reply:
Now https://github.com/cfs-energy/cfspopcon/issues/115